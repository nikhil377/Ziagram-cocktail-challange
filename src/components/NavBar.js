import React from 'react'
import { Link, withRouter }  from 'react-router-dom'
import * as img from '../images/LogoMain.png';

const Navbar =()=>{
    return(
      <nav className="navbar">
        <div className="container">
          <div className="navbar-brand">
            <Link to="/" className="navbar-item is-size-1"><img src={img} alt="Cocktail logo" /></Link>
          </div>
        </div>
      </nav>
    )
}

export default withRouter(Navbar)
