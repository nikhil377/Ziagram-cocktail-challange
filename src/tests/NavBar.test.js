// MyComponent.test.js
import React from 'react';
import NavBar from '../components/NavBar';
import { configure ,shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

configure({ adapter: new Adapter() });
let wrapper;
beforeEach(()=>{
    wrapper = shallow(<NavBar />);
})
describe("NavBar testcase", () => {
  test('should render the navbar', () => {
     expect(wrapper.getElements()).toMatchSnapshot();
});
});
  
