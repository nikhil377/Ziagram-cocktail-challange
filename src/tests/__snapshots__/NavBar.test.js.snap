// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar testcase should render the navbar 1`] = `
Array [
  <Context.Consumer>
    [Function]
  </Context.Consumer>,
]
`;
